save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPlugin.kt

```diff
@@ -70,6 +71,16 @@ class FixAndWarnPlugin(
 
         val fixTestResults = fixPlugin.handleFiles(files).toList()
 
+        val fixTestResultsFail = fixTestResults.filter { fixResult ->
+            fixResult.status !is Pass
+        }
+
+        val expectedFilesWithPass = expectedFiles.filter { expectedFile ->
+            fixTestResults.filter { fixResult ->
```

Comment:
You use here almost the same logic for filtering, I think it could be done in more clear way and in one step:

```kotlin
        val (fixTestResultsPassed, fixTestResultsFailed) = fixTestResults.partition { it.status is Pass }
```
Then you will be able to use `fixTestResultsPassed` in `expectedFiles.filter`
and `fixTestResultsFailed` for return statement

Reply:
Yes, thanks :)